ykcompile/src/lib.rs

```diff
@@ -458,18 +588,39 @@ impl TraceCompiler {
         self.asm.finalize().unwrap()
     }
 
+    #[cfg(test)]
```

Comment:
Can we add a comment saying why we did this. I know why, but let's help those who read our code later.

Reply:
There's a comment at https://github.com/softdevteam/yk/pull/74/commits/570d79148dabc4a0434b9eae164ccccaa4a1352f#diff-162e1dfdca14767450c768ad342d54f2R887 which explains this, but I can moved it up here if you think that's a better place.

Reply:
I think the additional comment would aid readability.

Reply:
Fixed in 11b078a